sr_robot_launch/launch/sr_hardware_control_loop.launch

```diff
@@ -1,4 +1,5 @@
 <launch>
+  <arg name="sim" default="true"/>
```

Comment:
I wouldnt name is as sim as it implies it uses software simulation like in other launch files. Maybe ursim?